src/config/conflists.c

```diff
@@ -35,54 +35,44 @@
 
 #include "config.h"
 #include "common.h"
+#include "log.h"
 
 #include <string.h>
 #include <glib.h>
 
 gboolean
 conf_string_list_add(GKeyFile* keyfile, const char* const group, const char* const key, const char* const item)
 {
+    if (!item) {
+        log_warning("Invalid parameter in `conf_string_list_add`: item is NULL. keyfile=%p, group=%s, key=%s", (void*)keyfile, group, key);
```

Comment:
I'm not sure `keyfile=%p` will be useful?

Reply:
makes sense.

Reply:
Addressed.